client/main.lua

```diff
@@ -338,26 +343,50 @@ RegisterNetEvent('QBCore:Client:VehicleInfo', function(data)
 end)
 
 RegisterNetEvent('lockpicks:UseLockpick', function(isAdvanced)
-    if cache.vehicle then
-        hotwire(isAdvanced)
+    local vehicle = cache.vehicle
+    if vehicle then
+        if isSearchAllowed then
+            setSearchLabelState(false)
+            hotwire(vehicle, isAdvanced)
+            setSearchLabelState(true)
+        end
     else
         lockpickDoor(isAdvanced)
     end
 end)
 
 RegisterNetEvent('qbx_vehiclekeys:client:OnLostKeys', function()
     Wait(0)
-    showHotwiringLabel()
+    showHotwiringLabel(cache.vehicle)
 end)
 
 AddEventHandler('ox_lib:cache:seat', function()
-    showHotwiringLabel()
+    showHotwiringLabel(cache.vehicle)
 end)
 
 AddEventHandler('ox_lib:cache:vehicle', function()
-    showHotwiringLabel()
+    showHotwiringLabel(cache.vehicle)
 end)
 
+lib.onCache('vehicle', function (vehicle) ---for some reason the autolock works with this
+end)
```

Comment:
This empty function is needed to make auto lock work?

Reply:
yes,
https://github.com/overextended/ox_lib/issues/618